cassandra-executor/src/main/java/com/mesosphere/dcos/cassandra/executor/backup/S3StorageDriver.java

```diff
@@ -172,152 +186,84 @@ public void upload(BackupRestoreContext ctx) throws IOException, URISyntaxExcept
             }
         }
 
+        tx.shutdownNow();
```

Comment:
Can we enclose the code that uses TransferManager into a try/catch/finally and perform tx.shutdownNow() in finally to avoid any leaks ?